config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -50,6 +52,8 @@ periodics:
     preset-e2e-scalability-common: "true"
   annotations:
     testgrid-alert-email: kubernetes-sig-scale@googlegroups.com,cloud-kubernetes-scalability-oncall@google.com
+    testgrid-dashboards: sig-release-master-blocking, sig-scalability-gce, google-gce
+    testgrid-tab-name: gce-master-scale-performance
```

Comment:
Please add a comment that OWNER: sig-scalability in description here and above

Reply:
Actually, maybe it doesn't make much sense, given that it's defined in this directory.